### ISSUE TITLE
track and detect dataset contamination

### ISSUE THREAD

**Issue by KennethEnevoldsen** [embeddings-benchmark/mteb#1636]
In multiple threads tracking dataset contamination have been mentioned as multiple models do not share their training dataset.

This issue is intended to link these discussions together as well as potential approaches to go about this issue.

- 
- LLM specific (probably won't work for embeddings)
  - Time Travel in LLMs: Tracing Data Contamination in Large Language Models ([ref](https://arxiv.org/abs/2308.08493))
- Overview
  - Leak, Cheat, Repeat: Data Contamination
and Evaluation Malpractices in Closed-Source LLMs ([ref](https://aclanthology.org/2024.eacl-long.5/))
- related work
  - [LM Contamination Index](https://hitz-zentroa.github.io/lm-contamination/)

**Comment by AdnanElAssadi** [comment]
@KennethEnevoldsen @Samoed any thoughts on whether we should try to tackle this as part of the moeb release?

**Comment by Samoed** [comment]
I don't think that this should be part of moeb, because this is not clear how to implement for now and need futher research
